.travis.yml

```diff
@@ -17,6 +17,17 @@ jobs:
     - env: LABEL=cppcheck
       script:
         - cppcheck --force --enable=warning,missingInclude --error-exitcode=1 . >build.log 2>&1 || (cat build.log && exit 1)
+# ppc64le support code
+    - env: LABEL=gcc
+      arch: ppc64le
+      compiler: gcc
+    - env: LABEL=cland
```

Comment:
There's a typo in the `env` value, shouldn't it be `LABEL=clang`?

Reply:
Hi @babelouest 
Yes correct I thought of updating it, since to keep similarities between amd64 and ppc64le have updated as it is. Let me know if changes can be made for both amd64 and ppc64le as LABEL=clang.

Reply:
I'm not sure we understand each other, it's just a simple typo in the `LABEL` value, it should be clan**g**, not clan**d**

Reply:
you can push the change in your branch, this PR will be automatically updated

Reply:
Sure will make the changes to this PR. Just to elaborate further, in AMD64 also there is a typo shall I update that code too? 
jobs:
  include:
    - env: LABEL=gcc
      compiler: gcc
    - env: LABEL=cland                        <<< This is how it is updated for amd64 
      compiler: clang
    - env: LABEL=cppcheck
      script:
        - cppcheck --force --enable=warning,missingInclude --error-exitcode=1 . >build.log 2>&1 || (cat build.log && exit 1)
# ppc64le support code
    - env: LABEL=gcc
      arch: ppc64le
      compiler: gcc
    - env: LABEL=cland                <<< This is for ppc64le 
      arch: ppc64le
      compiler: clang
    - env: LABEL=cppcheck
      arch: ppc64le
      script:
        - cppcheck --force --enable=warning,missingInclude --error-exitcode=1 . >build.log 2>&1 || (cat build.log && exit 1)

Reply:
> in AMD64 also there is a typo shall I update that code too?

Yes please

Reply:
Thanks. Have included those changes as part of this PR.